src/constants.js

```diff
@@ -70,9 +70,13 @@ const formats = {
 
 const intervals = {
   BLOCK: 20 * 1000,
+  COMPLETE_SYSTEM_WITHDRAWALS: 15 * 60 * 1000,
+  GAS_FRACTURE_NOTIFICATION: 15 * 60 * 1000,
```

Comment:
This isn't being used anywhere yet, but I guess it doesn't hurt to add it now, since we will need to support GAS break-up on mobile.